examples/mujoco_walking_position.ipynb

```diff
@@ -0,0 +1,325 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## (W.I.P.)Mujoco with IK and MPC Example \n",
+    "This examples, load a basic robot model (i.e. composed only of basic shapes), modifies the links of such a robot model by elongating the legs, define instances of the IK (Inverse Kinematic) and Centroidal MPC  controller and simulate the behavior of the robot using mujoco.  \n",
+    "\n",
+    ":warning: this is still a work in progress. Missing tuning on the controller to perfrom few steps "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Comodo import\n",
+    "from comodo.mujocoSimulator.mujocoSimulator import MujocoSimulator\n",
+    "from comodo.robotModel.robotModel import RobotModel\n",
+    "from comodo.robotModel.createUrdf import createUrdf\n",
+    "from comodo.centroidalMPC.centroidalMPC import CentroidalMPC\n",
+    "from comodo.centroidalMPC.mpcParameterTuning import MPCParameterTuning\n",
+    "from comodo.inverseKinematic.inverseKinematicParamTuning import (\n",
+    "    InverseKinematicParamTuning,\n",
+    ")\n",
+    "from comodo.inverseKinematic.inverseKinematic import InverseKinematic"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# General  import\n",
+    "import xml.etree.ElementTree as ET\n",
+    "import numpy as np\n",
+    "import tempfile\n",
+    "import urllib.request"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Getting stickbot urdf file and convert it to string\n",
+    "urdf_robot_file = tempfile.NamedTemporaryFile(mode=\"w+\")\n",
+    "url = \"https://raw.githubusercontent.com/icub-tech-iit/ergocub-gazebo-simulations/master/models/stickBot/model.urdf\"\n",
+    "urllib.request.urlretrieve(url, urdf_robot_file.name)\n",
+    "# Load the URDF file\n",
+    "tree = ET.parse(urdf_robot_file.name)\n",
+    "root = tree.getroot()\n",
+    "\n",
+    "# Convert the XML tree to a string\n",
+    "robot_urdf_string_original = ET.tostring(root)\n",
+    "\n",
+    "create_urdf_instance = createUrdf(\n",
+    "    original_urdf_path=urdf_robot_file.name, save_gazebo_plugin=False\n",
+    ")"
```

Comment:
I prefer to avoid it. I would like to encourage the usage of comodo with string objects rather than files.  I prefer not to give too many utilities for converting string model to file and from file to encourage to avoid an excessive use of such functions that could create big issues (passing from file several time )